### ISSUE TITLE
General Info

### ISSUE THREAD

**Issue by germacovitch** [olewolf/DHT_nonblocking#4]
Hi there
Are you using this lib? Any issues?
Br 
Gilberto

**Comment by olewolf** [comment]
Yes, I've used it myself on two similar projects, one of which is still active. I haven't encountered any problems.
